content/student/relax.md

```diff
@@ -0,0 +1,139 @@
+title: На досуге
+slug: relax
+category: student
+tags: [_nav]
+submenu_index: 8
+type: tag_by_slug_sidebar
```

Comment:
What are we going to show up in the sidebar of the page? I mean, do we really need the sidebar to be present?